lib/internal/test_runner/reporter/tap.js

```diff
@@ -31,13 +31,14 @@ async function * tapReporter(source) {
   yield `TAP version ${kDefaultTAPVersion}\n`;
   for await (const { type, data } of source) {
     switch (type) {
-      case 'test:fail':
+      case 'test:fail': {
         yield reportTest(data.nesting, data.testNumber, 'not ok', data.name, data.skip, data.todo);
-        yield reportDetails(data.nesting, data.details);
+        const location = `${data.file}:${data.line}:${data.column}`;
+        yield reportDetails(data.nesting, data.details, location);
         break;
-      case 'test:pass':
+      } case 'test:pass':
```

Comment:
```suggestion
      }
      case 'test:pass':
```

Reply:
I copied the existing style from this file (see line 57). I don't particularly like this style either though. Maybe we should add a lint rule in a separate change.